### ISSUE TITLE
Make it easier to select OSes

### ISSUE THREAD

**Issue by justanotheranonymoususer** [ergrelet/windiff#5]
Most of the time, I want to see only amd64, not arm, and only base updates, not kb. Not sure how to implement it, perhaps with a filter with options like there are options for symbol display, or perhaps selection which is an expandable tree, so I can collapse arm and expand each os to see kbs only if I need them.

**Comment by ergrelet** [comment]
Hi! 

Yes, there are improvements to be made on that front I think, not sure exactly how right now I'll try to think of something better.  
In the mean time, you can filter "base amd64" Windows versions by entering `amd64 (BASE)` in the combo box (but maybe you already knew that).

**Comment by ergrelet** [comment]
Yes indeed, I just pushed an update to use natural sorting instead so it should be better 👍

**Comment by ergrelet** [comment]
I'll consider this one fixed for now. Feel free to open another issue if you think of something else.
